cloudformation_templates/aws_dev_access.json

```diff
@@ -9,7 +9,7 @@
     },
     "CidrIp": {
       "Type": "String",
-      "Description": "CIDR or IP to use for security group rules"
```

Comment:
Looking at the docs it seems the or should go?
"CidrIp - Specifies a CIDR range."
 http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group-ingress.html#cfn-ec2-security-group-ingress-cidrip